frontend/apps/marketing/src/components/carousels/imageCarousel/ImageCarousel.tsx

```diff
@@ -0,0 +1,62 @@
+'use client';
+
+import '@code-dot-org/component-library/carousel/index.css';
+import React, {useMemo} from 'react';
+
+import DSCOCarousel, {
+  CarouselProps,
+} from '@code-dot-org/component-library/carousel';
+
+import Image from '@/components/image';
+import {ExperienceAsset} from '@/types/contentful/ExperienceAsset';
+
+export type ImageCarouselProps = {
+  /** Carousel content w/ fields from Contentful */
+  slides: ExperienceAsset[];
+  /** Number of slides to show at once */
+  slidesPerView?: Extract<CarouselProps['slidesPerView'], number>;
+};
+
+const ImageCarousel: React.FC<ImageCarouselProps> = ({
+  slides,
+  slidesPerView,
+}) => {
+  if (!slides) {
+    return (
+      <div style={{color: 'var(--text-neutral-primary)'}}>
+        <em>
+          <strong>🖼️ Image carousel placeholder.</strong> Please add a
+          "Carousel" content type entry in the Content sidebar, save, and open
+          the preview tab to see the carousel in action.
+        </em>
+      </div>
+    );
+  }
+
+  const slidesData = useMemo(
+    () =>
+      slides
+        .filter(slide => slide?.fields?.file?.url)
+        .map(({fields}) => {
+          const {title, description, file} = fields;
+
+          return {
+            id: title,
+            slide: <Image src={file?.url} altText={description} />,
```

Comment:
nit: `file` is known to be truthy at this point (because of the filter), so the `?` isn't needed

Reply:
I thought so too, but when I remove it I get an error: 
<img width="621" alt="Screenshot 2025-05-07 at 1 17 42 PM" src="https://github.com/user-attachments/assets/2e29b0cc-5ada-4d2c-8649-2f4c11c755fd" />